### ISSUE TITLE
browser sync does not work anymore

### ISSUE THREAD

**Issue by SamirJoza** [braginteractive/StrapPress#4]
For some odd reason browsersync dies out, when running gulp. All files appear to be processed but browsersync just does not get triggered. 
I am running: MacOS 10.14 (Mojave), Mamp Pro, and I use Node: v8.12.0 npm: 6.4.1

When I downloaded the theme a while back all was working fine (pre gulp 4).

Any help is greatly appreciated.

**Comment by braginteractive** [comment]
Do you have any errors? 

Everything is working for me. 

Node: 8.9.3
NPM: 6.3.0
Gulp: 4.0.0
Gulp CLI: 1.4.0

**Comment by SamirJoza** [comment]
No, no errors are being reported, it just hangs on Browser refresh.  (well hanging is not really correct: since SCSS and all watch tasks are being executed, only browser sync is not refreshing. 
gulp --version
[10:07:44] CLI version 2.0.1
[10:07:44] Local version 4.0.0
npm --version
6.4.1
node --version
v8.12.0
I'll try to "downgrade" gulp and npm and maybe that will do the trick. Thank you for your quick response

**Comment by braginteractive** [comment]
Hmm, not sure how to troubleshoot. 

If you are using the latest StrapPress file, don't downgrade Gulp. The gulpfile is using the latest syntax.
